src/main/webapp/chart.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Charts Project(Part 1)</title>
+    <!-- take in core charts package -->
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+      google.charts.load('current', {packages: ['corechart']});
+      google.charts.setOnLoadCallback(drawChart);
+      function drawChart(){
+        // create a new DataTable instance
+        var book_data = new google.visualization.DataTable();
+
+        // define columns for the DataTable instance
```

Comment:
Right now you are loading dummy data, AFAICT. Please make this explicit here in the comment and add TODO for fixing this and loading data at runtime from somewhere, like this:

// Fill the table with static example data
// TODO: Load and update the data at runtime.

Reply:
Added the TODO comment